data-claims-event-service/src/main/resources/application.yml

```diff
@@ -16,6 +16,7 @@ spring:
 logging:
   level:
     uk.gov.justice.laa.dstew.payments.claimsevent: ${APPLICATION_LOGGING_LEVEL:INFO}
+    io.sentry: DEBUG
```

Comment:
As a reminder, you may wish to remove this before merging into may, in order to reduce noise.